cs-multi-shell/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,46 @@
+import { ProviderOptions } from '@ionic-enterprise/auth';
+
 export const environment = {
   production: true
 };
+
+export const urlSchemeIonicAuthOptions: ProviderOptions = {
```

Comment:
This is where we keep out settings for the auth provider for native and for web.